pkg/gendoc/markdown.go

```diff
@@ -60,7 +60,7 @@ __Version:__ {{ .Version }}
 
```

Comment:
can you change the comment to accurately reflect the contents of the commit? "updated generated files: documentation, API etc."